packages/common/test/projects/HubProjects.test.ts

```diff
@@ -359,7 +359,7 @@ describe("HubProjects:", () => {
       // verify the query
       const searchOpts = searchSpy.calls.argsFor(0)[0];
 
-      expect(searchOpts.q).toBe(`type:"Hub Project" AND water`);
+      expect(searchOpts.q).toBe(`(type:"Hub Project") AND (water)`);
```

Comment:
This looks kinda weird but I did some manual tests agains the portal api and verified that it handles it ok with the term (water) wrapped in parens.